<launch>
  <rosparam command="load" ns="shelf_detection" file="$(find shelf_detection)/config/ros.yaml"/>
  <!--<rosparam command="load" ns="shelf_detection" file="$(find shelf_detection)/config/face_recognition.yaml"/>!-->

  <node name="shelf_detection_node" pkg="shelf_detection" type="shelf_detection_node.py" output="screen"/>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/butia_vision/sd/shelf_detection_view"/>
    <param name="autosize" value="true" />
  </node>
</launch>
